### ISSUE TITLE
rust build

### ISSUE THREAD

**Issue by lazovicff** [vercel/vercel#2087]
When running `now dev`:

> Ready! Development server running at http://localhost:3000
> Running initial builds
downloading files
downloading the rust toolchain
installing openssl-devel...
failed to `yum install -y openssl-devel`
> Error! spawn yum ENOENT

MacOS 10.14.3 (18D109)
now.json: 
```
{
    "version": 2,
    "builds": [{
        "src": "/api/score/src/main.rs",
        "use": "@now/rust"
    }],
    "routes": [{
        "src": "/api/score",
        "methods": ["POST"],
        "dest": "/api/score/src/main.rs",
        "headers": {
            "Access-Control-Allow-Origin": "*"
        }
    }]
}

```

**Comment by maciek134** [comment]
Same on Arch. `$ uname -a`:
```
Linux #### 5.0.0-arch1-1-ARCH #1 SMP PREEMPT Mon Mar 4 14:11:43 UTC 2019 x86_64 GNU/Linux
```
`$ now --version`:
`15.0.3`

**Comment by exul** [comment]
A dirty hack is to create a symlink for `yum` that points to `echo` so that `yum install -y openssl-devel` returns with exit code 0.
I have `~/bin` in my `PATH`, after running `ln -s /bin/echo $HOME/bin/yum` `now dev` works under Arch Linux. You'll have to install openssl for your OS manually.

**Comment by goughjo02** [comment]
I have hit this problem on macbook High Sierra, could anyway please help out?

**Comment by jacobrosenthal** [comment]
(Random lurker here) If you notice that Merged issue linking here, that should fix it. Were just waiting for a new release probably in the next few days on https://github.com/zeit/now-cli/releases that includes it

**Comment by goughjo02** [comment]
Thank you.

**Comment by jacobrosenthal** [comment]
Fixed for me as of cli 15.5.0

**Comment by TooTallNate** [comment]
The Rust builder is now being community-maintained, so please try out the [`now-rust`](https://github.com/mike-engel/now-rust) builder and open an issue on that repository if you still have any issues.
